iam.tf

```diff
@@ -55,27 +60,30 @@ data "aws_iam_policy_document" "assume-role-policy" {
     actions = ["sts:AssumeRole"]
 
     principals {
-      type        = "AWS"
-      identifiers = ["${format("arn:aws:iam::%s:root", data.aws_caller_identity.current.account_id)}"]
+      type = "AWS"
+      identifiers = [format(
```

Comment:
This is short enough to be a one liner. IMO.

Reply:
This doesn't really fit on one line nicely, so I think it is probably more readable as is.